From 4324ee59a379017e2f43983146673351b9ac9152 Mon Sep 17 00:00:00 2001
From: Avinash Kumar <avinash3.kumar@intel.com>
Date: Mon, 29 Sep 2025 18:04:38 +0000
Subject: [PATCH] Fix android build issue with new kernel 6.12.47

Patch 0016-TWL-Fix-kernel-config-kernel-6.12.40-errors.patch not needed.
Its already defined prior with new kernel. Removing that.
Path 0004-Update-kernel-defconfig-6.12.patch is modified only with needed defines.

Tracked-On: OAM-134158
Signed-off-by: Avinash Kumar <avinash3.kumar@intel.com>
---
 .../gmin64/config-lts/linux-intel-lts2024/x86_64_defconfig     | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/groups/kernel/gmin64/config-lts/linux-intel-lts2024/x86_64_defconfig b/groups/kernel/gmin64/config-lts/linux-intel-lts2024/x86_64_defconfig
index a4de199b..903e0765 100644
--- a/groups/kernel/gmin64/config-lts/linux-intel-lts2024/x86_64_defconfig
+++ b/groups/kernel/gmin64/config-lts/linux-intel-lts2024/x86_64_defconfig
@@ -6126,6 +6126,7 @@ CONFIG_RTLLIB_CRYPTO_WEP=m
 #
 # Android
 #
+CONFIG_ANDROID=y
 CONFIG_ANDROID_STAGING=y
 CONFIG_ASHMEM=y
 CONFIG_ASHMEM_C=y
@@ -8021,7 +8022,7 @@ CONFIG_BOOTPARAM_HARDLOCKUP_PANIC=y
 CONFIG_DETECT_HUNG_TASK=y
 CONFIG_DEFAULT_HUNG_TASK_TIMEOUT=120
 CONFIG_BOOTPARAM_HUNG_TASK_PANIC=y
-# CONFIG_DETECT_HUNG_TASK_BLOCKER is not set
+CONFIG_DETECT_HUNG_TASK_BLOCKER=y
 # CONFIG_WQ_WATCHDOG is not set
 # CONFIG_WQ_CPU_INTENSIVE_REPORT is not set
 # CONFIG_TEST_LOCKUP is not set
-- 
2.34.1

